VR/VRFramebuffer.cpp

```diff
@@ -0,0 +1,543 @@
+#include "VRFramebuffer.h"
+
+/************************************************************************************
+
+Original file name  : XrCompositor_NativeActivity.c
+
+Copyright : Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.
+
+*************************************************************************************/
```

Comment:
Hmm, wonder if this license thing is something to worry about with this file? Though it looks like straightforward stuff, and if this is meant to just be a template for people to edit, I think we can remove this comment?

Reply:
The code is taken from SDK sample code. There shouldn't be anything to worry about. I removed it.